cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -503,16 +493,17 @@ func shouldExpose(ing *networkingv1.Ingress) bool {
 	return isTSIngress && pgAnnot != ""
 }
 
-func (a *IngressPGReconciler) getVIPService(ctx context.Context, name tailcfg.ServiceName, logger *zap.SugaredLogger) (*VIPService, error) {
+func (a *IngressPGReconciler) tryGetVIPService(ctx context.Context, name tailcfg.ServiceName) (_ *VIPService, exists bool, _ error) {
```

Comment:
I would prefer to keep the existing setup where the error type indicates 'not found' rather than return a bool. The existing setup is consistent with kube client (both the one from c/r and the internal one). We could make the error check more similar too (`IsNotFound(err)` etc). That would just make it easier to read these control loops - rather than having multiple ways how the same thing is done